src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2861,9 +2861,9 @@ public override IEnumerable<ISymbol> GetSymbolsWithName(Func<string, bool> predi
 
         #endregion
 
-        internal override AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerManager analyzerManager)
+        internal override AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerManager analyzerManager, bool reportDiagnosticsWithSourceSuppression)
```

Comment:
This flag will probably go away as it will be part of CompilationOptions and doesn't need to be retained by the analyzer driver.